Command/LoadFixtureCommand.php

```diff
@@ -135,7 +135,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($dirOrFile) {
             $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
         } else {
-            $paths = array();
+            /** @var $kernel \Symfony\Component\HttpKernel\KernelInterface */
```

Comment:
i prefer to use a short class name here and `use` it at the top if its not already used.